components/routes/RoutesList.js

```diff
@@ -2,19 +2,25 @@ import React, { useEffect, useState } from 'react';
 import { View, Text, FlatList } from 'react-native';
 import DropDownPicker from 'react-native-dropdown-picker';
 import styles from '../../styles/Route.style';
+import { FontAwesome, FontAwesome6, MaterialIcons, Octicons } from '@expo/vector-icons';
+import { AntDesign } from '@expo/vector-icons';
```

Comment:
```suggestion
import { FontAwesome, FontAwesome6, MaterialIcons, Octicons, AntDesign  } from '@expo/vector-icons';
```

These can be combined